airqo-platform/AirQo-api
# Add node affinity

**_WHAT DOES THIS PR DO?_**
- Adds node affinity for all deployments and cronjobs

**_WHAT ISSUES ARE RELATED TO THIS PR?_**
- Jira cards
    - [PLAT-1252](https://airqoteam.atlassian.net/browse/PLAT-1252?atlOrigin=eyJpIjoiOGM3YjdmMzdhY2FmNDdkMmJmYzQ4OTI2ZDQyZGIwMWIiLCJwIjoiaiJ9)
    - [PLAT-1254](https://airqoteam.atlassian.net/browse/PLAT-1254?atlOrigin=eyJpIjoiOTAwNThiZTcyYjk3NDIwMzgwMGJlZTBlNGVlZDRjOGYiLCJwIjoiaiJ9)

**_HOW DO I TEST OUT THIS PR?_**
- Apply any of manifests to create a deployment/cronjob.
- Take note of the respective node affinity
- Confirm that the deployment/cronjob is running on the specified node by running the command below
    `kubectl get pods --output=wide`

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
- n/a

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
- n/a

**_ARE THERE ANY RELATED PRs?_**
- [This frontend PR](https://github.com/airqo-platform/AirQo-frontend/pull/787)
